src/tools/opt-dist/src/bolt.rs

```diff
@@ -65,8 +65,13 @@ pub fn bolt_optimize(path: &Utf8Path, profile: &LlvmBoltProfile) -> anyhow::Resu
         .arg("-jump-tables=move")
         // Fold functions with identical code
         .arg("-icf=1")
+        // The following flag saves about 50 MiB of libLLVM.so size.
+        // However, it succeeds very non-deterministically. To avoid frequent artifact size swings,
+        // it is kept disabled for now.
+        // FIXME(kobzol): try to re-enable this once BOLT in-place rewriting is merged or after
+        // we bump LLVM.
```

Comment:
@Kobzol has this been reevaluated yet?

Reply:
Yes, I tried it a few weeks ago:
![image-6.png](https://github.com/user-attachments/assets/8ea25219-8459-43b4-91ac-457fb7d58a5c)

Sadly, it's still quite unstable. Asked the BOLT people about it, but we haven't got anywhere yet.